### DISCUSSION TITLE
Does git-crypt use its own version of git?

### DISCUSSION THREAD

**Discussion by elliottcrif** [AGWA/git-crypt#244] [General]
Does git-crypt use its own version of git?

**Comment by AGWA** [comment]
No, it uses whatever version of Git is installed on the system.

**Reply by elliottcrif** [reply]
👍 I am seeing a weird error that made me think that was occurring. For context, I can manually run git status on the CI server and not get a fatal error, but when git-crypt attempts to unlock I get this error. @AGWA Any idea why this would occur?
```
  fatal: unsafe repository ('/opt/teamcity-agent-1/work/95435a27928575d8' is owned by someone else)
  To add an exception for this directory, call:
  git config --global --add safe.directory /opt/teamcity-agent-1/work/95435a27928575d8
  git-crypt: Error: 'git status' failed - is this a Git repository?
```

git-crypt 0.6
git 2.25

**Reply by nmalaguti** [reply]
I ran into a similar issue which turned out to be caused by https://github.com/git/git/blob/8ddf593a250e07d388059f7e3f471078e1d2ed5c/Documentation/RelNotes/2.36.0.txt#L16-L23

Adding `git config --global --add safe.directory '*'` as a step in my CI script was able to address the issue.

**Reply by elliottcrif** [reply]
I can confirm that adding that git config fixes the issue.
